tycho-apitools-plugin/src/main/java/org/eclipse/tycho/apitools/ApiAnalysisMojo.java

```diff
@@ -162,6 +162,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 			} catch (Exception e) {
 				throw new MojoFailureException("Can't fetch dependencies!", e);
 			}
+			System.setProperty("DetectVMInstallationsJob.disabled", "true");
```

Comment:
This can be too late, not sure at which point in time this code is triggered...

Reply:
This code is triggered (job is scheduled) when the jdt.launching bundle starts, which can be pretty early as it's required by jdt.core to resolve the classpath IIRC.
Wouldn't it be possible to set the flag before starting the VM (or at least the Equinox container) ?

Reply:
1. This *is* before Equinox Container starts
2. I now using the preference instead (so no unforeseen side effects)
3. I have debugged that this prevents the job from being started.

So if setting it on the Jenkins Job directly works as system-property, this should then work here as well.